fares-data-build-tool/fdbt-site
# feat: Added lineId to be passed through to matching JSON

# Description

-   Added lineId to the matching JSON for all journeys which use services (single, return, flatFare, periodMultiService and multiOperatorMultiService).

# Testing instructions

-   Run through site locally for all of the above ticket types and log out the matchingJson which gets created, and ensure the lineId is included for all lines where it is present in database (only Blackpool 2C has NULL).

# Type of change


# Checklist:
